frontend/src/app/matching/matching.component.ts

```diff
@@ -27,19 +33,44 @@ export class MatchingComponent {
     this.getQueueLength()
   }
 
+  stopTimer() {
+    this.timer.s = 0;
+    this.timer.mn += 1;
+    clearInterval(this.countDown)
+    this.countDown = undefined
+  }
+
   getMatch(userDetails: any) {
+    // this.countDown = setInterval(() => {
```

Comment:
Should this be removed?

Reply:
That is actually the UI timer for one of the assignments. I commented it out because I need Justin's help to check and debug it first.